src/mono/mono/mini/mini-windows.c

```diff
@@ -189,6 +189,7 @@ mono_runtime_install_handlers (void)
 	win32_seh_set_handler(SIGFPE, mono_sigfpe_signal_handler);
 	win32_seh_set_handler(SIGILL, mono_crashing_signal_handler);
 	win32_seh_set_handler(SIGSEGV, mono_sigsegv_signal_handler);
+	win32_seh_set_handler(SIGTERM, mono_sigterm_signal_handler);
```

Comment:
If this is going to work fully on Windows we also need to handle that signal in win32_seh_set_handler as well as react to the exception that will be generated and passed to seh_vectored_exception_handler.

Reply:
Adding the `case SIGTERM` and the corresponding `term_handler` variable in `exceptions_[x86|amd64]` appears straightforward but how are they then used? I see there is exception handling for `EXCEPTION_ILLEGAL_INSTRUCTION` etc. where the handler is picked up and called, but how and where would this particular signal be fielded?

Reply:
@lateralusX @lambdageek Looking for further guidance on the outstanding Windows' issue.